fastq_filtrator.py

```diff
@@ -0,0 +1,64 @@
+def gc(read, gc_bounds=(0, 100)): #the function for GC filtering
+    seq = read[1].rstrip('\n') #geting rid of '\n' at the end of the string
+    C = seq.count('C') #counting of Cytosine in the read
+    G = seq.count('G') #counting of  Guanin in the read
+    CG = round((((int(C) + int(G)) / (len(seq) + 1)) * 100), 1) #counting of GC proportion
```

Comment:
Это хорошо читается, но не очень оптимально по производительности. count внутри себя делает полный проход по строке, и мы вызываем его два раза, хотя GC можно посчитать за один проход. Но это мелочи)

Все стандартные переменные в питоне принято называть маленькими буквами, кроме классов и глобальных констант. Даже если это аббревиатуры

Reply:
Ещё не понятно зачем округлять ГЦ-состав и прибавлять к длине 1
```suggestion
    C = seq.count('C') #counting of Cytosine in the read
    G = seq.count('G') #counting of  Guanin in the read
    CG = (C + G) / len(seq) * 100 #counting of GC proportion
```